sparkmagic/sparkmagic/utils/configuration.py

```diff
@@ -175,7 +175,10 @@ def default_samplefraction():
 def pyspark_sql_encoding():
     return u'utf-8'
     
-    
+@_with_override
+def pyspark_python_encoding():
```

Comment:
Hmm... I don't think it's needed. If you want, you can call it `dataframe_python_encoding`. Clarity is OK, but I don't see how independence is ever needed for this, do you?